integration-cli/docker_cli_daemon_test.go

```diff
@@ -1888,8 +1888,13 @@ func (s *DockerDaemonSuite) TestDaemonNoSpaceleftOnDeviceError(c *check.C) {
 	err := s.d.Start("--graph", "/tmp/testfs-mount")
 	c.Assert(err, check.IsNil)
 
+	reg := setupRegistry(c, false)
+	defer reg.Close()
+	_, err = setupImage(c)
+	c.Assert(err, check.IsNil)
+
 	// pull a repository large enough to fill the mount point
-	out, err := s.d.Cmd("pull", "registry:2")
+	out, err := s.d.Cmd("pull", repoName)
 	c.Assert(out, check.Not(check.Equals), 1, check.Commentf("no space left on device"))
```

Comment:
Please see: https://github.com/docker/docker/pull/19484#discussion_r50270648
Dup PR?  But either way, I'd like to see the error check changed.